aten/src/ATen/test/vec_test_all_types.cpp

```diff
@@ -1135,6 +1146,75 @@ namespace {
             if (AssertVectorized<vec>(NAME_INFO(Quantize), expected, actual).check()) return;
         } //trials;
     }
+#if !defined(CPU_CAPABILITY_DEFAULT) && !defined(_MSC_VER)
```

Comment:
Suggest to make it specific to avx256 and avx516 here.
```suggestion
#if (defined(CPU_CAPABILITY_AVX2) ||  defined(CPU_CAPABILITY_AVX512)) && !defined(_MSC_VER)
```

Reply:
Thanks for advice, done.